README.md

```diff
@@ -1,26 +1,50 @@
-# Package size calculator
+# Package Size Calculator
 
-Simple tool to calculate the size of a NPM package. It installs the package in a Docker container so any malicious packages won't affect your system.
+A simple tool to calculate the size of an NPM package. This tool installs the package in a Docker container to ensure that any potentially malicious packages do not affect your system.
 
-## Using
+## Installation
 
-You can find the built binaries [here](https://github.com/TheDevMinerTV/package-size-calculator/releases).
+You can download the pre-built binaries from the [Releases page](https://github.com/TheDevMinerTV/package-size-calculator/releases).
 
-> :warning: `package-size-calculator` requires having a Docker daemon running.
+## Prerequisites
+
+> :warning: `package-size-calculator` requires a running Docker daemon.
+
+### Installing Docker
+
+Follow the official Docker installation guides for your platform:
+
+- [Docker Desktop for Windows](https://docs.docker.com/desktop/install/windows-install/)
+- [Docker Desktop for Mac](https://docs.docker.com/desktop/install/mac-install/)
+- [Docker Engine for Linux](https://docs.docker.com/desktop/install/linux-install/#generic-installation-steps)
+
+## Usage
+
+To calculate the size of an NPM package, run:
 
 ```bash
-package-size-calculator
+package-size-calculator <package-name>
```

Comment:
The CLI doesn't accept a package name parameter.

Reply:
Done.